### DISCUSSION TITLE
Παραδοτέο 2ο

### DISCUSSION THREAD

**Discussion by VasilisG11** [courses-ionio/hci#2005] [Show and tell]
# **Ακολουθεί σχετικό asciinema έπειτα απο την εγκατάσταση των arch σε flash drive**

[![asciicast](https://asciinema.org/a/q6dcJI4M1Kf5sWtqTvU6rwb7O.svg)](https://asciinema.org/a/q6dcJI4M1Kf5sWtqTvU6rwb7O)

[Η Αναφορά μου](https://github.com/VasilisG11/hci/tree/2019024/projects/2019024)

**Comment by YGNT7777** [comment]
Καλησπέρα Βασίλη,
Γενικά ωραία το έχεις κάνει.
Όμως για να βαθμολογιθεί η άσκηση
θα πρέπει στο promt σου να έχει το ΑΜ σου
για παράδειγμα μπορεί να είναι vasilis@2019024
Το αλλάζεις αυτο στο /.bashrc

source: https://courses-ionio.github.io/help/cli/
